integration/cursors/getmore_test.go

```diff
@@ -952,45 +957,6 @@ func TestCursorsGetMoreCommandMaxTimeMSCursor(t *testing.T) {
 		integration.AssertMatchesCommandError(t, mongo.CommandError{Code: 50, Name: "MaxTimeMSExpired"}, cursor.Err())
 	})
 
-	t.Run("FindGetMoreMaxTimeMS", func(tt *testing.T) {
```

Comment:
Those 2 test cases seem unnecessary. They mean to be resolved in #2983, which is about maxTimeMS propagation, however usage of maxTimeMS for non-awaitdata cursors is impossible, and it's resolved and tested in this PR